x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/index.tsx

```diff
@@ -61,7 +63,12 @@ export function WaterfallWithSummary({
 
   const { entryWaterfallTransaction } = waterfall;
 
-  if ((!entryWaterfallTransaction || traceSamples.length === 0) && !isLoading) {
+  if (
+    !entryWaterfallTransaction &&
+    traceSamples.length === 0 &&
+    !isLoading &&
+    isNotInitiated
```

Comment:
So when the request is not initiated the not found message is displayed?

Reply:
Good catch, I'll fix it